src/features/events/hooks/useEventContact.ts

```diff
@@ -0,0 +1,38 @@
+import useEventMutations from '../hooks/useEventMutations';
+import useEventParticipantsData from './useEventParticipantsData';
+import useEventParticipantsMutations from './useEventParticipantsMutations';
+
+type useEventContacReturn = {
+  removeContact: () => void;
+  setContact: (contactId: number) => void;
+};
+
+export default function useEventContact(
```

Comment:
Nice! I like how this hook does a very specific thing, but then uses other hooks to do (part of) the heavy lifting.